source/includes/administration/_notifications.md

```diff
@@ -0,0 +1,364 @@
+            "updatedAt": "2020-10-29T19:28:22.000Z"
+        },
+        "splash": false,
+        "updatedAt": "2020-10-29T19:27:34.462Z",
+        "status": "published"
+    }
+}
+```
+Get a specific notification.
+
+Attributes | &nbsp;
+---------- | -----------
+`id`<br/>*UUID* | The id of the notification.
+`authorId`<br/>*UUID* | The id of the notification's author.
+`authorFirstName`<br/>*string* | The notification's author's first name.
+`authorLastName`<br/>*string* | The notification's author's last name.
+`splash`<br/>*boolean* | When set to true, the notification will be displayed in the home screen of CloudMC until it reaches its deprecated date. Only one of splash and sticky can be set to true.
+`sticky`<br/>*boolean* | When set to true, the notification will be displayed upon logging into CloudMC. Only one of splash and sticky can be set to true.
+`status`<br/>*string* | The status of the notification.
+`createdAt`<br/>*string* | The date on which the notification was created.
+`updatedAt`<br/>*string* | The date on which the notification was updated.
+`publishedAt`<br/>*string* | The date on which the notification was published.
+`depricatedAt`<br/>*string* | The date on which the notification will be deprecated.
+`version`<br/>*integer* | The notification's version.
+`category`<br/>*Array[[Notification Category](#administration-notification-categories)]* | The notification category that the notification belongs to.<br/>*includes*: `id`, `organizationId`, `createdAt`, `updatedAt`, `translations`, `icon`, `version`
+`translations`<br/>*Array[Object]* | The translations of the notification category's title.
+`translations.language`<br/>*string* | The language of that the translation is in.
+`translations.text`<br/>*string* | The content of the notification.
+`translations.type`<br/>*string* | The content that we are translating. (Must be one of: title, body, description, url slug)
+`translations.id`<br/>*UUID* | The translation's id.
+`translations.version`<br/>*integer* | The translation's version
+
+<!-------------------- CREATE NOTIFICATION CATEGORY -------------------->
+
+#### Create notification category
+
+`POST /content/categories/`
+
+```shell
+# Create notification category
+curl -X POST "https://cloudmc_endpoint/v2/rest/content/categories/" \
+   -H "MC-Api-Key: your_api_key" \
+   -H "Content-Type: application/json" \
+   -d "request-body"
+```
+> Request body example:
+
+```json
+{
+    "translations": [
+        {
+            "language": "en",
+            "text": "system-notification-create-no-author",
+            "type": "title"
+        },
+        {
+            "language": "fr",
+            "text": "en français",
+            "type": "title"
+        },
+        {
+            "language": "en",
+            "text": "system notification create",
+            "type": "body"
+        },
+        {
+            "language": "fr",
+            "text": "en français",
+            "type": "body"
+        }
```

Comment:
is there a validation on the types that are required for translation (body,title,etc)?

Reply:
If one of those types is not specified, an invalid field error message is returned by the API. In the java code, the translation's type is marked as an enum with those four values specified

Reply:
i guess it would be a good idea to mention it in the doc. I checked and it is just validated when you set the notification as published

Reply:
Add it to the description at line 279

Reply:
Added to the field description 65d3185551293faa443a5e59617d81963fb934f1